shared/js/ui/views/privacy-practices.es6.js

```diff
@@ -0,0 +1,44 @@
+const ParentSlidingSubview = require('./sliding-subview.es6.js')
+const SiteModel = require('./../models/site.es6.js')
+const overviewTemplate = require('./../templates/shared/privacy-practices-overview.es6.js')
+const detailsTemplate = require('./../templates/shared/privacy-practices-details.es6.js')
+
+function PrivacyPractices (ops) {
+    this.model = null
+    this.currentModelName = null
+    this.template = ops.template
+
+    ParentSlidingSubview.call(this, ops)
+
+    this._cacheElems('.js-privacy-practices', [
+        'overview',
+        'details',
+    ])
+
+    this.setupClose();
+    this.renderAsyncContent();
+}
+
+PrivacyPractices.prototype = $.extend({},
+    ParentSlidingSubview.prototype,
+    {
+        renderAsyncContent: function () {
+            const random = Math.round(Math.random()*100000)
+            this.currentModelName = 'site' + random
+
+            this.model = new SiteModel({
+                modelName: this.currentModelName
+            })
+
+            this.model.getBackgroundTabData().then(() => {
+                this.$overview.html(overviewTemplate(
+                    this.model.domain,
+                    this.model.tosdr
+                ))
+                this.$details.html(detailsTemplate(this.model.tosdr))
```

Comment:
Again, I looked at the `GradeDetails` view as an example when putting this together. I decided to handle things in a different way, though. The pattern used there (use the template twice, once to render the view, once as a thing to append) was really unclear to me at first. It seems unnecessarily coupled to the implementation of the view, and when the model is created.

What do you think to this approach?

Reply:
Yeah its a better pattern for sure. Alot of the UI was thrown together quite quickly, improvements welcome.